platform-dom/src/main/java/com/softicar/platform/dom/input/DomSelect.java

```diff
@@ -62,6 +62,16 @@ public final O getSelectedOption() {
 		}
 	}
 
+	/**
+	 * Returns a list of all selected option objects.
```

Comment:
```suggestion
	 * Returns all selected option objects.
```
...it's not quite a "list".

Reply:
did so